code/modules/paperwork/ticketmachine.dm

```diff
@@ -102,14 +103,18 @@
 	handle_maptext()
 
 /obj/machinery/ticket_machine/proc/handle_maptext()
-	switch(ticket_number) //This is here to handle maptext offsets so that the numbers align.
-		if(0 to 9)
-			maptext_x = 13
-		if(10 to 99)
-			maptext_x = 10
-		if(100)
-			maptext_x = 8
-	maptext = "<font face='Small Fonts'>[ticket_number]</font>"
+	if(!dispense_enabled)
+		maptext_x = 13
+		maptext = "<font face='Verdana'>X</font>"
```

Comment:
```suggestion
		maptext = "<font face='Small Fonts' color='#ff0000'>X</font>"
```
We use `Small Fonts` for **ALL** maptext because it doesnt anti alias. The current X looks like a dent in the screen. Making it red would also stand out a lot better.

Reply:
I decided changing the text to 'OFF' would be better than a 'X', and thought it being a very dark gray would make it not stand out so much as a bright red.